solitude/utils.py

```diff
@@ -2,6 +2,22 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
+# Note deliberately ignoring o, O, 0, i and 1 as too similar.
+choices = 'abcdefghjklmnpqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ23456789'
+
+
+def shorter(integer):
+    """
+    Convert an integer into something shorter.
+    """
+    output = ''
+    while integer > 0:
+        remainder = integer % len(choices)
+        output = choices[remainder] + output
```

Comment:
Sure although how much of an improvement it is anymore it certainly has changed over the Python versions. https://gist.github.com/andymckay/8a93550b725b1abba270. http://stackoverflow.com/questions/1349311/python-string-join-is-faster-than-but-whats-wrong-here is a good read.